### ISSUE TITLE
Natuzora Kanata: File could not be opened as resource archive

### ISSUE THREAD

**Issue by JACK-CLDS** [morkt/GARbro#527]
I try to unpack Natuzora Kanata(夏空彼方) but not successful.
Double click will show "无法作为压缩包打开文件 (File could not be opened as resource archive)", and if I extract a file, this file will be extracted but not opened correctly.

**Comment by Cosetto** [comment]
> Natuzora Kanata

use Krkrextract

**Comment by JACK-CLDS** [comment]
Yes, I used Krkrextract. I think this is an issue that needs feedback.

**Comment by Cosetto** [comment]
It is because Natuzora Kanata has encryption and you have to choose the right encryption for it in order to open it.
